src/controllers/HighLightedText.js

```diff
@@ -0,0 +1,55 @@
+import models from '../db/models';
+import errorResponse from '../helpers';
+
+const { HighlightedText: HighLightedTextModel, Article } = models;
+/**
+ * @description class representing comment on Highlightedtext Controller
+ * @class HighLightedText
+ */
+class HighLightedText {
+  /**
+   * @description - This method is responsible for creating new articles
```

Comment:
hi @okoroemeka your description says this method is for creating new articles

Reply:
Nope. Thanks for pointing that out. I will fix it now.

Reply:
Fixed